setup.sh

```diff
@@ -19,6 +19,12 @@ scram project ${CMSSWVER}
 cd ${CMSSWVER}/src/
 # cmsenv
 eval `scramv1 runtime -sh`
+git clone https://gitlab.cern.ch/DeepAK8/NNKit.git
+cp NNKit/misc/mxnet_predict.xml $CMSSW_BASE/config/toolbox/$SCRAM_ARCH/tools/selected
+scram setup mxnet_predict
+rm $CMSSW_BASE/external/$SCRAM_ARCH/lib/libmxnet_predict.so
```

Comment:
it seems like the `rm` isn't necessary, since the `cp` on the next line will automatically overwrite the existing library.